Integrations/SignalSciences/SignalSciences.yml

```
@@ -0,0 +1,692 @@
+    - name: ip
+      required: true
+      description: IP Address to be added.
+    - name: note
+      required: true
+      description: Note associated with the tag.
+    - name: expires
+      description: Optional RFC3339-formatted datetime in the future. Omit this paramater
+        if it does not expire.
+    outputs:
+    - contextPath: SigSciences.Blacklist.Source
+      description: IP Address present in the Blacklist.
+      type: string
+    - contextPath: SigSciences.Blacklist.Note
+      description: Note associated with the tag.
+      type: string
+    - contextPath: SigSciences.Blacklist.ID
+      description: ID for this particular entry.
+      type: string
+    - contextPath: SigSciences.Blacklist.CreatedBy
+      description: User who added this source.
+      type: string
+    - contextPath: SigSciences.Blacklist.CreatedDate
+      description: Created Date Timestamp.
+      type: date
+    - contextPath: SigSciences.Blacklist.ExpiryDate
+      description: Expiration Timestamp.
+      type: date
+    description: Add an IP to the blacklist residing on Signal Sciences' Platform.
+  - name: sigsci-whitelist-remove-ip
+    arguments:
+    - name: siteName
+      required: true
+      description: 'Please input the siteName you want see whitelist for. e.g : siteName='
+    - name: IP
+      required: true
+      description: IP Address entry to be removed.
+    description: Remove an IP from whitelist residing on Signal Sciences' Platform.
+  - name: sigsci-blacklist-remove-ip
+    arguments:
+    - name: siteName
+      required: true
+      description: 'Please input the siteName you want see blacklist for. e.g : siteName='
+    - name: IP
+      required: true
+      description: IP Address entry to be removed.
+    description: Remove an IP from blacklist residing on Signal Sciences' Platform.
+  - name: sigsci-get-sites
+    arguments: []
+    outputs:
+    - contextPath: SigSciences.Sites.Name
+      description: Site name
+      type: string
+    - contextPath: SigSciences.Sites.CreatedDate
+      description: Site creation date
+      type: date
+    description: Get all the site name from SignalSciences
+  - name: sigsci-create-site-list
+    arguments:
+    - name: siteName
+      required: true
+      description: The name of the site in Signal Sciences you want to add a list
+        to
+    - name: list_name
+      required: true
+      description: The name for the list you wish to create on Signal Sciences
+    - name: list_type
+      required: true
+      description: The type for the list you wish to create on Signal Sciences. Legal
+        types are IP, String, Country or Wildcard
+    - name: entries_list
+      required: true
+      description: A list of values, consistent with the list's type, separated by
+        commas
+      isArray: true
+    - name: description
+      description: Short text which describes your newly created list
+    outputs:
+    - contextPath: SigSciences.SiteLists.Name
```

I think it would be better
```suggestion
    - contextPath: SigSciences.Site.List.Name
```

---

I disagree on this one.
There are many sitelists for a specific site, and each has several own attributes.
It's just like the standard in many other integrations with things like "Elements", "Countries" etc